ee-feature-pack/ee-10-api/src/main/resources/modules/system/layers/base/javax/batch/api/main/module.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<!--
-  ~ Copyright 2020 Red Hat, Inc.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~   http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-
-<module-alias xmlns="urn:jboss:module:1.9" name="javax.batch.api" target-name="jakarta.batch.api"/>
```

Comment:
Please restore this. These aliases are a form of compatibility, as all of these modules were public API.

Perhaps we should get rid of them, but if we do I think it should be a general discussion and then we get rid of all of them at once.

Reply:
`javax.batch.api` module.xml restored.